source/_integrations/technove.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: TechnoVE
+description: Instructions on how to integrate TechnoVE Charging station with Home Assistant.
+ha_category:
+  - Car
+ha_release: 2024.1
+ha_iot_class: Local Polling
+ha_domain: technove
+ha_config_flow: true
+ha_codeowners:
+  - '@Moustachauve'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The TechnoVE integration provides connectivity with TechnoVE WiFi Charging station through the station local API.
```

Comment:
Yes it should be the same. There is a non-wifi charging station available but it doesn't have any connectivity options so there is not really multiple types supported.

I removed the "WiFi" part since it is implied only the WiFi model can be connected. Let me know if I should do the opposite to make it more obvious to users.

Reply:
hm... In the first sentence, you say the non-Wi-Fi option cannot be connected. Then, would it be incorrect to add the WiFi to the name? Since it is the only one that can be connected? As it is now, I'm unsure if the documentation only mentions the WiFi option, but there is an option that was forgotten in the documentation.
Personally, if it is not incorrect to add WiFi to the name, I'd find that clearer.

Reply:
Yes, sure. I just verified and it seems like the branding used by TechnoVE for the WiFi model is "Smart Charging Station", so I renamed it to that so that it can fit the real product's name.